source/guide_bitwarden.rst

```diff
@@ -0,0 +1,370 @@
+ * Type ``nightly`` as default toolchain as it is required for bitwarden_rs.
+ * Type ``default`` to use the default install profile. 
+ * Enter ``y`` to add rust  to the PATH.
+ * Enter ``1`` to proceed with the installation
+
+.. code-block:: console
+ :emphasize-lines: 16,21,24,27,30,42
+
+ [isabell@stardust ~]$ curl https://sh.rustup.rs -sSf | sh
+ info: downloading installer
+
+ Welcome to Rust!
+ [...]
+ Current installation options:
+ 
+   default host triple: x86_64-unknown-linux-gnu
+     default toolchain: nightly
+               profile: default
+  modify PATH variable: yes
+
+ 1) Proceed with installation (default)
+ 2) Customize installation
+ 3) Cancel installation
+ >	2
+
+ I'm going to ask you the value of each of these installation options.
+ You may simply press the Enter key to leave unchanged.
+
+ Default host triple?
+
+ Default toolchain? (stable/beta/nightly/none)
+	nightly
+
+ Profile (which tools and data to install)? (minimal/default/complete)
+	default
+
+ Modify PATH variable? (y/n)
+	y
+
+ Current installation options:
+
+   default host triple: x86_64-unknown-linux-gnu
+     default toolchain: nightly
+               profile: default
+  modify PATH variable: yes
+
+ 1) Proceed with installation (default)
+ 2) Customize installation
+ 3) Cancel installation
+ >	1
+ [...]
+ Rust is installed now. Great!
+
+
+To finish the setup, **logout and login** again or run
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ source $HOME/.cargo/env
+
+Install Bitwarden_rs
+--------------------
+
+Clone the repository into your document root. It will create the directory ``~/bitwarden_rs`` automatically.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://github.com/dani-garcia/bitwarden_rs.git
+ 
+In order to build bitwarden_rs  sucessfully you'll need to set an environment variable pointing to the sqlite3 header files:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ export SQLITE3_LIB_DIR=/var/lib64
+
+``cd`` into the automatically created folder:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd bitwarden_rs
+ [isabell@stardust bitwarden_rs]$ 
+
+Build the server executable:
+
+.. note :: If that doesn't work the first time and the build failed, **just try again**.
+
+.. code-block:: console
+
+ [isabell@stardust bitwarden_rs]$ cargo build --release --features sqlite
+
+In the next step we will download the latest build for the web vault. Check `this page`_ for the newest build number and **replace it** in the following snippet:
+
+.. note :: If you don't want to use the web-vault feature (web-app access to your vaults) for any security reasons you can skip this step. Add the following line to your ``env`` file later if you do so:
+  ``WEB_VAULT_ENABLED=false``
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust bitwarden_rs]$ [isabell@stardust bitwarden_rs]$ wget https://github.com/dani-garcia/bw_web_builds/releases/download/v2.11.0/bw_web_v2.13.2.tar.gz
```

Comment:
Two different version numbers in single link

Reply:
there are also 2 prompts in one line